appserver/ejb/ejb-http-remoting/client/src/main/java/fish/payara/ejb/http/client/Lookup.java

```diff
@@ -45,7 +45,7 @@
 import java.util.Base64;
 import java.util.Map;
 
-abstract class Lookup {
+public abstract class Lookup {
```

Comment:
Why was it necessary to raise visibility of this class?

Reply:
Good catch! Earlier it's internal method was used in the Base auth client request filter.